rviz_common/src/rviz_common/properties/property.cpp

```diff
@@ -251,14 +253,8 @@ void Property::setParent(Property * new_parent)
 
 QVariant Property::getViewData(int column, int role) const
 {
-  if (role == Qt::ForegroundRole &&
-    ( parent_ && parent_->getDisableChildren() ) )
-  {
-#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
-    return Qt::gray;
-#else
-    return QColor(Qt::gray);
-#endif
+  if (role == Qt::TextColorRole && parent_ && parent_->getDisableChildren()) {
```

Comment:
The Windows build is complaining that `Qt::TextColorRole` is deprecated.
[Docs suggest changing this to Qt::ForegroundRole](https://doc.qt.io/qt-5/qt.html#ItemDataRole-enum).

Reply:
Looks right to me on the Mac - want retriever the other tests?